tests/unit/constraints/test_base.py

```diff
@@ -496,10 +496,10 @@ def test_reverse_transform(self):
         """
         # Run
         instance = Constraint()
-        output = instance.reverse_transform('input')
+        output = instance.reverse_transform(pd.DataFrame())
 
         # Assert
-        assert output == 'input'
+        pd.testing.assert_frame_equal(output, pd.DataFrame())
```

Comment:
maybe add a small check to see that it is a different object from what was passed in (like compare id or something)